src/main/java/com/alkemy/ong/model/Organization.java

```diff
@@ -0,0 +1,58 @@
+package com.alkemy.ong.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.hibernate.annotations.Where;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "organization")
+@Getter
+@Setter
+
+@SQLDelete(sql = "UPDATE organization SET softDelete = true WHERE id=?")
+@Where(clause = "softDelete = false")
+
+public class Organization implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @Column(name = "id", nullable = false)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
```

Comment:
Tanto en este atributo, como en todos los deben ser NOT NULL falta agregar el mensaje de error, eso lo hacemos mediante una annotation también. Podrías agregar lo siguiente:
`@NotEmpty(message = "Name can not be empty")` por ejemplo